<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>线程池</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addTaskButton">
         <property name="text">
          <string>添加任务</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearQueueButton">
         <property name="text">
          <string>清空队列</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>线程数:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minThreadSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxThreadSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>任务队列</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>等待执行的任务:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="taskQueueList"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>线程状态</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>工作线程:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="workingThreadList"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>空闲线程:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="idleThreadList"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>统计信息</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="totalThreadsLabel">
         <property name="text">
          <string>总线程:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="busyThreadsLabel">
         <property name="text">
          <string>忙线程:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="idleThreadsLabel">
         <property name="text">
          <string>空闲线程:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="queueSizeLabel">
         <property name="text">
          <string>任务队列:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="completedTasksLabel">
         <property name="text">
          <string>完成任务:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>运行日志</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="logTextBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
